batchflow/models/tf/mobilenet.py

```diff
@@ -189,38 +232,112 @@ def body(cls, inputs, name='body', **kwargs):
                 for k in range(repeats):
                     if k > 0:
                         block['strides'] = 1
-                    x = cls.block(x, **block, residual=k > 0, name='block-%d' % i, **kwargs)
+                    residual = block.pop('residual', False) or k > 0
+                    x = cls.block(x, **block, residual=residual, name='block-%d' % i, **kwargs)
```

Comment:
block and kwargs can contain same parameters

Reply:
fixed.